web_src/js/index.js

```diff
@@ -668,7 +668,7 @@ function initIssueComments() {
     event.preventDefault();
     updateIssuesMeta(
       url,
-      isChecked === 'true' ? 'attach' : 'detach',
+      isChecked === 'true' ? 'detach' : 'attach',
```

Comment:
How come you've switched these around? 

I'm not saying it's wrong but to understand this I need to look in 3 places - and I don't completely understand what's happened here - could you explain?

Reply:
Beacause ``is-checked`` means it has been requested to requst, to follow previous logic in ``updateIssuesMeta``, so attach means add and detach means remove, so if it's checked , should remove , otherwise need add, previous logic is wrong.

Reply:
Cool I thought that was what was happening. Thanks.